runtime/src/main/resources/application.properties

```diff
@@ -45,4 +45,5 @@ otel.exporter.otlp.enabled=false
 spring.activiti.history-level=full
 git.command.timeout.seconds=10
 http.request.retry.max.attempts=5
-http.request.retry.delay.milliseconds=1000
\ No newline at end of file
+http.request.retry.delay.milliseconds=1000
+caffeine.cache.expire.time=60
```

Comment:
The `caffeine.cache.expire.time` is used in two different class, one used time unit is second but another is minute.If the time of these two configurations is different, it is better to use two different configuration items.